test/integration/legacy_json_integration_test.cc

```diff
@@ -69,10 +69,6 @@ TEST_P(LegacyJsonIntegrationTest, TestServerGrpcJsonTranscoder) {
   createTestServer("test/config/integration/server_grpc_json_transcoder.json", {"http"});
 }
 
-TEST_P(LegacyJsonIntegrationTest, TestServerRatelimit) {
```

Comment:
Any particular reason to remove from the legacy integration config tests?

Reply:
Just mopping up the JSON, we can keep it there if you prefer. I think we have sufficient coverage from the proto-based test.

Reply:
It's not a big deal. I still feel like we are losing some ease of inspection of what is actually being tested without being able to scan the full configs, but I will get over it.